TestFoundation/TestNSData.swift

```diff
@@ -206,7 +206,6 @@ class TestNSData: XCTestCase {
             ("test_dataHash", test_dataHash),
             ("test_genericBuffers", test_genericBuffers),
             ("test_writeFailure", test_writeFailure),
-            ("testBasicConstruction", testBasicConstruction),
```

Comment:
why was this test removed?

Reply:
It was a duplicate line, that test is called on line 198

Reply:
ah ok, I see it past the code fold